tests/test_main.py

```
@@ -0,0 +1,49 @@
+
+import curses
+from game import constants
+
+
+class TestGameLoop(unittest.TestCase):
+    """Test the main game loop functionality."""
+
+    @patch('time.sleep')
+    @patch('time.time')
+    @patch('curses.wrapper')
+    def test_game_initialization(self, mock_wrapper, mock_time, mock_sleep):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_game_initialization`

Add return type annotation: `None`

(ANN201)